Containerfile

```diff
@@ -25,7 +25,9 @@ COPY --from=planner /build/recipe.json recipe.json
 RUN cargo chef cook --recipe-path recipe.json
```

Comment:
Don't you want to also specify a target here since this builds dependencies?

Reply:
Wont that mean `shuttle-${folder}` will be compiled to wasm32-wasi and put in `target/wasm32-wasi`, which we don't actually want? Only the project, like the axum-wasm example, should be compiled to wasm32-wasi, and all the wasi stuff will be in there hidden by the codegen. But the deployer needs the target added so it can compile wasm32-wasi.

Reply:
Sorry my bad, I meant profile (`--release`), not target.

Reply:
Ah, of course, that makes sense. Yes, we should do that, I'll add it there too.